### DISCUSSION TITLE
Welcome to pymoo Discussions!

### DISCUSSION THREAD

**Discussion by blankjul** [anyoptimization/pymoo#184] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cyrilpic** [comment]
Hi Julian,
Great idea to try out this new feature of GH. 

So, I am Cyril, a research on design automation for mechanical systems. I used pymoo to perform constrained multi-objective optimization and contributed to some of the test problems.

I think pymoo is one of the cleanest and most practical package to develop, use and test evolutionary algorithms.
